src/typings.d.ts

```diff
@@ -18,4 +18,6 @@ declare var customElements: CustomElements;
 
 declare interface CustomElements {
     define: any;
-}
\ No newline at end of file
+}
+
+interface Window { ClarityIcons: any; }
```

Comment:
Yes. As long as, that's how they would use `ClarityIcons.add()` after loading `clarity-icons.min.js`.